lti_consumer/utils.py

```diff
@@ -46,3 +46,18 @@ def get_lms_lti_launch_link():
     return u"{lms_base}/api/lti_consumer/v1/launch/".format(
         lms_base=get_lms_base(),
     )
+
+
+def validate_preflight_response(response):
```

Comment:
@viadanna Since this is an LTI 1.3 specific validator, it'll need to be put inside the `lti_1p3` folder.
You can create a new `utils.py` file there for any support functions, as the goal of that module is to be reusable in other scenarios later on, and easily extractable from the current codebase.